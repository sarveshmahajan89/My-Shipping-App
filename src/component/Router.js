import React from "react";
import Home from "./Home.js"
import About from "./About.js"
import Contact from "./Contact.js"

import { Switch, Route } from 'react-router-dom';

class Router  extends React.Component {
    render() {
        return (
            <header className="masthead">
                <div className="container">
                    <div className="intro-text">
                        <div className="content" >
                            <Switch>
                                <Route exact path='/' component={Home} />
                                <Route path='/about' component={About} />
                                <Route path='/contact' component={Contact} />
                            </Switch>
                        </div>
                    </div>
                </div>
            </header>
        );
    }
}
export default Router ;