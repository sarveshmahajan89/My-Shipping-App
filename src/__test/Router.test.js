import React from 'react';

import Router from '../component/Router.js'

it('Router renders correctly', () => {

    expect(<Router />).toMatchSnapshot();
});
