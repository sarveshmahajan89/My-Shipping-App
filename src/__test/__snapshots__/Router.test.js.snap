// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router renders correctly 1`] = `<Router />`;
